wire-ios/Wire-iOS/Sources/AppRootRouter.swift

```diff
@@ -220,7 +222,7 @@ extension AppRootRouter: AppStateCalculatorDelegate {
                          completion: completionBlock)
         case .locked:
             // TODO: [John] Avoid singleton.
```

Comment:
I guess one way it would be to pass it to this locked associated value like for authenticated case - but probably not in the scope of this PR

Reply:
Yeah, it's coming in another PR.